postgres/tests/test_slru.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2022-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+import pytest
+
+from datadog_checks.postgres.util import SLRU_METRICS
+
+from .common import POSTGRES_VERSION
+
+
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+@pytest.mark.skipif(
+    POSTGRES_VERSION is None or float(POSTGRES_VERSION) < 13.0,
+    reason='SLRU test requires version 13.0 or higher (make sure POSTGRES_VERSION is set)',
+)
```

Comment:
We shouldn't skip the test for versions <13, we should just change the assertions.

Reply:
Since I've made the metric a default now, I've moved the tests to the `test_common_metrics`

Also, to check that we really don't generate those values for version < 13, I've added the `aggregator.assert_all_metrics_covered` if that's okay.